src/app/components/shared/baw-client/baw-client.component.ts

```diff
@@ -0,0 +1,81 @@
+import { Component, ElementRef, OnInit, ViewChild } from "@angular/core";
+import { DomSanitizer, SafeResourceUrl } from "@angular/platform-browser";
+import { NavigationEnd, Router } from "@angular/router";
+import { withUnsubscribe } from "@helpers/unsubscribe/unsubscribe";
+import { assetRoot } from "@services/config/config.service";
+import { interval } from "rxjs";
+import { filter, takeUntil } from "rxjs/operators";
+
+@Component({
+  selector: "baw-client",
+  template: `
+    <baw-loading *ngIf="loading"></baw-loading>
+
+    <iframe *ngIf="url" #content [src]="url" (load)="updateIframeSize()">
+      <!-- This warning only shows on browsers which don't support iframes -->
+      <p>
+        Unfortunately your browser does not support iframes. Please ensure you
+        are utilising a common browser which is running the most up to date
+        version.
+      </p>
+    </iframe>
+  `,
+  styles: [
+    `
+      iframe {
+        width: 100%;
+        min-height: 100%;
+        border: 0px;
+      }
+    `,
+  ],
+})
+export class BawClientComponent extends withUnsubscribe() implements OnInit {
+  @ViewChild("content") private iframeRef: ElementRef<HTMLIFrameElement>;
+  public url: SafeResourceUrl;
```

Comment:
this should probably be an`@input` so we can customise what baw-client page is loaded